clients/go/pkg/commands/activateJobs_command.go

```diff
@@ -116,16 +107,14 @@ func (cmd *ActivateJobsCommand) Send() ([]entities.Job, error) {
 	return activatedJobs, nil
 }
 
-func NewActivateJobsCommand(gateway pb.GatewayClient, requestTimeout time.Duration, retryPredicate func(context.Context, error) bool) ActivateJobsCommandStep1 {
+func NewActivateJobsCommand(gateway pb.GatewayClient, retryPredicate func(context.Context, error) bool) ActivateJobsCommandStep1 {
```

Comment:
Just a small thing, wouldn't it be clearer to have a type here instead of a raw function signature for the retry predicate?

Reply:
Yes! I think we discussed it when the retry logic was introduced but I'm not sure what the argument was. Maybe it was because like this it's clearer what you have to pass to it, rather than if it's just a type retryPredicate. But I agree it should be refactored, I can do it in this issue since you mention it